custom_components/easee/services.py

```diff
@@ -1012,6 +1028,50 @@ async def charger_execute_set_access(call):
             f"Could not find charger: {call.data.get(CHARGER_ID, 'Unknown')}"
         )
 
+    async def charger_execute_set_ocpp(call):
+        """Execute a service with an action command to Easee charging station."""
+
+        charger = await async_get_charger(call)
+        enable = call.data.get(ATTR_ENABLE)
+        url = call.data.get(ATTR_OCPP_URL)
+
+        if url is None:
+            url = "ws://127.0.0.1:9000"
+
+        _LOGGER.debug("Call set ocpp enable %d on charger_id: %s with url %s",
+            enable,
+            charger.id,
+            url
+        )
+        if charger:
+            function_name = SERVICE_MAP[call.service]
+            function_call = getattr(charger, function_name["function_call"])
+            try:
+                return await function_call(enable, url)
```

Comment:
Currently, the pyeasee ocpp set function does not enable the config, so there needs to be an additional call to "apply" it.